src/components/Quiz.js

```diff
@@ -31,7 +34,11 @@ const Quiz = ({ questions, weights, updateWeights, setIsQuizComplete }) => {
       ...selectedChoices,
       [currentQuestionIndex]: selectedChoice,
     });
-    setAnsweredQuestions(answeredQuestions.map((answered, index) => index === currentQuestionIndex ? true : answered));
+
+    console.log(updatedWeights);
```

Comment:
Wouldn't typically be left in a prod environment but I'll let it slide here.

Reply:
Oops, thanks for the catch. I didn't mean to leave that in.